common/src/types.rs

```diff
@@ -31,6 +30,27 @@ pub struct Member {
     // - Unlock-If-The-Validator-Set-Changes
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct FinalizationSignTarget {
+    pub block_hash: Hash256,
+    pub round: ConsensusRound,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct FinalizationProof {
+    pub round: ConsensusRound,
```

Comment:
What does `ConsensusRound` mean? And should the treasury contract hold this as storage?

Reply:
In short, `ConsensusRound` is a value that is decided during the consensus process.
All you need to do is to understand the type of `FinalizationProof` has been changed, which is the type of `BlockHeader::prev_block_finalization_proof` and `proof` in `update_treasury_light_client`.